### ISSUE TITLE
v1.4.0 - npm run build leads to a failed build due to missing `unit-http` dependency

### ISSUE THREAD

**Issue by Thesephi** [Thesephi/fullstack-typescript-webapp-starter#5]
Unsure how this behaves on v1.3.0 and lower. But on v1.4.0, if we do `npm run build`, it runs for a while then crashes like so:

```
ERROR in ./node_modules/somnus/lib/somnus.js 2:887514-887534
Module not found: Error: Can't resolve 'unit-http' in '/workspaces/fullstack-typescript-webapp-starter/node_modules/somnus/lib'
resolve 'unit-http' in '/workspaces/fullstack-typescript-webapp-starter/node_modules/somnus/lib'
  Parsed request is a module
  using description file: /workspaces/fullstack-typescript-webapp-starter/node_modules/somnus/package.json (relative path: ./lib)
    resolve as module
      /workspaces/fullstack-typescript-webapp-starter/node_modules/somnus/lib/node_modules doesn't exist or is not a directory
      /workspaces/fullstack-typescript-webapp-starter/node_modules/somnus/node_modules doesn't exist or is not a directory
      /workspaces/fullstack-typescript-webapp-starter/node_modules/node_modules doesn't exist or is not a directory
      looking for modules in /workspaces/fullstack-typescript-webapp-starter/node_modules
        single file module
          using description file: /workspaces/fullstack-typescript-webapp-starter/package.json (relative path: ./node_modules/unit-http)
            no extension
              /workspaces/fullstack-typescript-webapp-starter/node_modules/unit-http doesn't exist
            .ts
              /workspaces/fullstack-typescript-webapp-starter/node_modules/unit-http.ts doesn't exist
            .tsx
              /workspaces/fullstack-typescript-webapp-starter/node_modules/unit-http.tsx doesn't exist
            .js
              /workspaces/fullstack-typescript-webapp-starter/node_modules/unit-http.js doesn't exist
        /workspaces/fullstack-typescript-webapp-starter/node_modules/unit-http doesn't exist
      /workspaces/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
 @ ./src/server/main.ts 54:31-48

webpack 5.73.0 compiled with 1 error in 14770 ms
```

**Comment by Thesephi** [comment]
`v1.3.0` and below used `somnus@3.x` which didn't have `unit-http` integration. Therefore, it's highly likely that versions from `v1.3.0` and below do not suffer from this issue. Ideally, this needs to be resolved in the `somnus` library, not here in this project.

**Comment by Thesephi** [comment]
Confirmed that this issue is resolved with `somnus@8.4.0`.
